api-spec/protobuf/tdex-daemon/v2/feeder.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
```

Comment:
NIT: In other protos all comments end with a `.`, we could maintain this convention in these new protos

Reply:
Done